src/api/SFA.DAS.Provider.Events.Api.IntegrationTestsV2/BeforeAllTests.cs

```diff
@@ -12,6 +12,29 @@ public class BeforeAllTests
         public async Task Setup()
         {
             TestData.EmployerAccountId = new Random().Next(100000000, 999999999);
+
+            TestData.CollectionPeriod = (byte)new Random().Next(1, 14);
+            
+            var year = (short) new Random().Next(2016, 2021);
```

Comment:
This seems overly complex for a test. Why not something like:
`["1819", "1920", "2021"]` and then `Random.Next(3)` into the array? It's much clearer in terms of the output and what you are trying to do.

Reply:
Agreed, but tests are passing and it would be lot of rework for little gain

Reply:
What's the point of a pull request then? 
There is confusing code in place that could be made simpler. Tests are supposed to be documentation and should be as simple as possible to understand. There's not much work and the gain would be significant.

Reply:
changed